.github/workflows/revert-to-previous-version-action.yml

```diff
@@ -0,0 +1,78 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+# graphic logo are (registered/a) trademark(s) of Plan International.
+name: Revert to a previous version
+
+on: 
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: Environment to revert
+        required: true
+        default: config
+        options:
+          - config
+          - staging
+      replicas:
+        description: No of replicas deployed in the environemt
+        default: "1"
+        required: true
+      version-tag:
+        type: choice
+        description: The version tag to revert back to
+        required: true
+        default: v1.0.1
+        options:
+          - v1.0.1
+jobs:
+  deploy:
+    uses: ./.github/workflows/deploy.yml
+    with:
+      environment: ${{ inputs.environment }}
+      core-image-tag: ${{ inputs.version-tag }}
+      farajaland-image-tag: ${{ inputs.version-tag }}
+    secrets:
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      SSH_KEY: ${{ secrets.SSH_KEY }}
+      KNOWN_HOSTS: ${{ secrets.KNOWN_HOSTS }}
+      SMTP_PORT: ${{ secrets.SMTP_PORT }}
+      SMTP_HOST: ${{ secrets.SMTP_HOST }}
+      SMTP_USERNAME: ${{ secrets.SMTP_USERNAME }}
+      SMTP_PASSWORD: ${{ secrets.SMTP_PASSWORD }}
+      ALERT_EMAIL: ${{ secrets.ALERT_EMAIL }}
+      DOCKERHUB_ACCOUNT: ${{ secrets.DOCKERHUB_ACCOUNT }}
+      DOCKERHUB_REPO: ${{ secrets.DOCKERHUB_REPO }}
+      KIBANA_USERNAME: ${{ secrets.KIBANA_USERNAME }}
+      KIBANA_PASSWORD: ${{ secrets.KIBANA_PASSWORD }}
+      MONGODB_ADMIN_USER: ${{ secrets.MONGODB_ADMIN_USER }}
+      MONGODB_ADMIN_PASSWORD: ${{ secrets.MONGODB_ADMIN_PASSWORD }}
+      ELASTICSEARCH_SUPERUSER_PASSWORD: ${{ secrets.ELASTICSEARCH_SUPERUSER_PASSWORD }}
+  restore-data:
+    needs: deploy
+    environment: ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+    env:
+      SSH_HOST: farajaland-${{ inputs.environment }}.opencrvs.org
+      REPLICAS: ${{ inputs.replicas }}
+      VERSION_TAG: ${{ inputs.version-tag }}
+    steps:
+      - name: Install SSH Key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.SSH_KEY }}
+          known_hosts: ${{ secrets.KNOWN_HOSTS }}
+      - name: Restore previous version data
+        run: ssh root@$SSH_HOST "MONGODB_ADMIN_USER=$MONGODB_ADMIN_USER MONGODB_ADMIN_PASSWORD=$MONGODB_ADMIN_PASSWORD ELASTICSEARCH_ADMIN_USER=$ELASTICSEARCH_ADMIN_USER ELASTICSEARCH_ADMIN_PASSWORD=$ELASTICSEARCH_ADMIN_PASSWORD /opt/opencrvs/infrastructure/emergency-restore-metadata.sh $VERSION_TAG $REPLICAS"
```

Comment:
Going to remove this script to write it again post v1.1.0 release as the path /opt/opencrvs would never work for v1.0.1 and also the Ansible scripts must run.

Reply:
Yeah it won't be usable pre-v1.1.0. So we may as well not keep it for now